### PR TITLE
Missing trainer sprite addition

### PR DESCRIPTION
**Pull request by IanWalder**
Most of these are Alola trainers. Prioritizing sprites that are free to use as long as they're credited

### CODE REVIEW

**Review by RedSparr0w — APPROVED** [review]
👍 Nice work, thanks for the PR! 🎉 

I also noticed [Gnomowladny](https://www.deviantart.com/gnomowladny) has quite a few of the Galar trainer sprites and are free to use with credit.
(we are already using their 'Team Flare Bryony' sprite)

File: src/modules/Credits.ts [hunk 1]
```diff
@@ -10,6 +10,30 @@ type Credit = {
 
 // Sort alphabetically
 export const SpriteCredits: Credit[] = [
+    {
+        name: 'Beliot419',
+        link: 'https://www.deviantart.com/beliot419',
+        image: 'https://a.deviantart.net/avatars-big/b/e/beliot419.png'
```
Comment (RedSparr0w) [inline src/modules/Credits.ts:16]: Minor fixup
```suggestion
        image: 'https://a.deviantart.net/avatars-big/b/e/beliot419.png',
```

**Comment by IanWalder** [comment]
That's all I got in me for now. In terms of Galar trainers I could only find Piers in Gnomowladny's gallery

**Comment by RedSparr0w** [comment]
Nice work, thanks 👍

**Pull request merged**